.github/workflows/e2e_test.yaml

```diff
@@ -109,6 +112,7 @@ jobs:
             --config path=${{ secrets.E2E_TESTING_REPO }} \
             --config token=${{ secrets.E2E_TESTING_TOKEN }} \
             --config virtual-machines=1 \
+            --config denylist=10.0.0.0/8 \
```

Comment:
Does it cover host_ip which is included in the denylist range ?

Reply:
Yes, this is intended to test the capability of the charm to resolve firewall conflicts.